src/Aspire.ProjectTemplates.9.0.net9/templates/aspire-empty/.template.config/ide.host.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://json.schemastore.org/ide.host",
+  "icon": "ide/AspireEmpty.ico",
+  "displayOverviewPage": "0",
+  "disableHttpsSymbol": "NoHttps",
+  "unsupportedHosts": [
+    {
+      "id": "vs",
+      "version": "(,17.9)"
```

Comment:
Does this need to be bumped higher? Are we doing any testing of Aspire 9 with VS 17.10 or 17.11?